addons/Assessments_Navigation_Bar/src/presenter.js

```diff
@@ -1697,11 +1775,15 @@ function AddonAssessments_Navigation_Bar_create(){
 
         var parsedState = JSON.parse(state);
         var upgradedState = presenter.upgradeState(parsedState);
+        const previousLeftSideIndex = parsedState.leftSideIndex;
+        const previousLeftSideValue = parsedState.leftSideValue;
```

Comment:
Jak czytam kod to nie powinno nic wybuchnąć w przypadku podania starego stanu, gdzie tych wartości nie ma.
(undefined < 0) == false
oraz gdy previousLeftSideValue == undefined to także nie znajdzie tej strony.

Proponuję jednak, aby w nowych metodach była lepsza obsługa na przypadek undefined niż warunki if z wartością undefined. Ewentualnie nadanie nowej wartości, gdyby previousLeftSideIndex i previousLeftSideValue były puste. 

Czy były wykonywane testy, przynajmniej manualne z starym stanem ?